docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM alpine:latest as build
+
+ARG PKG_MANAGER="yarn"
+ARG INSTALL_COMMAND="yarn install --pure-lockfile --production"
+
+RUN mkdir /opt/armadietto
+WORKDIR /opt/armadietto
+
+RUN apk add nodejs $PKG_MANAGER
+
+COPY package.json ./
+COPY package-lock.json ./
```

Comment:
This is probably unnecessary (since yarn is used), but it doesn't hurt.

Reply:
Yeah, from the suggested changes to the file. Thought it may be useful to leave in. Shouldn't affect the build.